hotel/src/hil/common.rs

```diff
@@ -0,0 +1,25 @@
+/// Possible errors returned by syscalls. In case of failure, the negative value of the error is
+/// returned to the application.
+#[derive(Copy, Clone)]
+pub enum SyscallError {
```

Comment:
The conversation has begun on the mailing list: [Here](https://groups.google.com/forum/#!topic/tock-dev/CjD_-YvDoQ4)

Once a standard error set has been decided, it should be pretty straightforward to fix the existing instances of SyscallError.